packages/serverless-deploy-iam/bin/app.ts

```diff
@@ -175,7 +175,7 @@ export class ServiceDeployIAM extends cdk.Stack {
                     {
                          name: 'IAM',
                          prefix: `arn:aws:iam::${accountId}:role`,
-                         qualifiers: [`${serviceName}*`],
+                         qualifiers: [`*${serviceName}*`],
```

Comment:
This doesn't seem right? :thinking: Why do we need this change?

Reply:
Yeah it was for addressing this error (Note `Cognito-` in the role name):
```
Resource handler returned message: "User: arn:aws:sts::<redacted>:assumed-role/<serviceName>-deploy-iam-ServiceRolev128EB9C79-2NHYkk36Zi32/AWSCloudFormation is not authorized to perform: iam:GetRole on resource: role Cognito-<serviceName>-stg-UnAuth because ....
```

Reply:
Fixed